File: left/pyramid.vb
'Visual Basic code
'Made in an online compiler
Module VBModule
    Sub Main()
        Dim height as integer
        
        Console.Write("Enter height of the pyramid: ")
        height = Console.ReadLine()
        
        For i as integer = 1 to height
            For j as integer = 1 to i
                Console.Write("*")
            Next
            Console.WriteLine()
        Next
    End Sub
End Module

File: left/pyramid.cs
//C#
using System;

class Pyramid
{
    public static void Main(String[] args)
    {
        Console.WriteLine("Enter height of pyramid: ");
        int h = int.Parse(Console.ReadLine());
        for(int i = 1; i <= h; i++)
        {
            for(int j = 1; j <= i; j++)
            {
                Console.Write("*");
            }
            Console.WriteLine();
        }
    }
}
